core/components/molecules/pagination-toolbar/pagination-toolbar.js

```diff
@@ -55,11 +44,14 @@ const StyledPageSelector = styled.div`
 `
 
 const Pager = ({ onPrevPressed, onNextPressed }) => (
-  <div>
-    {/* TODO: Remove usage of '<' and '>' characters, replace with proper icons. */}
-    <StyledPagerButton onClick={onPrevPressed}>{'<'}</StyledPagerButton>
-    <StyledPagerButton onClick={onNextPressed}>{'>'}</StyledPagerButton>
-  </div>
+  <ButtonGroup compressed>
+    <StyledPagerButton size="compressed" onClick={onPrevPressed}>
+      <Icon name="chevron-left" />
+    </StyledPagerButton>
+    <StyledPagerButton size="compressed" onClick={onNextPressed}>
+      <Icon name="chevron-right" />
+    </StyledPagerButton>
+  </ButtonGroup>
```

Comment:
Do you want it to look like two connected buttons?

![image](https://user-images.githubusercontent.com/1863771/43391273-ddb54f96-940d-11e8-9b93-06e61d9bb4b8.png)

This looks like it could be a single button with an icon

![image](https://user-images.githubusercontent.com/1863771/43391317-05211b8c-940e-11e8-9e4a-798dc3e3b62d.png)

@landitus what do you think?